gdrive/settings.py

```diff
@@ -13,6 +13,7 @@
 DEBUG = os.getenv("DEBUG", "False") == "True"
 
 LOG_LEVEL = os.getenv("LOG_LEVEL", logging.getLevelName(logging.INFO))
+logging.basicConfig(level=LOG_LEVEL)
```

Comment:
That is one way to do it but I feel like this line belongs in main and not in settings.

Reply:
I cant think of a way to get the log statements in settings to run without putting the basicConfig call here. `LOG_LEVEL` would need to be set and `logger.basicConfig()` called before `from gdrive import settings` is called to keep the `basicConfig` call in main.

Reply:
@dzaslavskiy thoughts?

Reply:
keep the settings import in main so that settings is executed at the begining.